code/game/objects/items/devices/scanners.dm

```diff
@@ -202,13 +202,13 @@ MASS SPECTROMETER
 			else
 				user << "<span class='info'>Blood level [blood_percent] %, [C.blood_volume] cl, type: [blood_type]</span>"
 
-		var/implant_detect
+		var/cyberimp_detect

```

Comment:
pet peeve of mine, cyber implant is terrible fucking name given that these are organs and not obj/item/weapon/implant.